tests/any_test_help.py

```diff
@@ -225,9 +225,10 @@ def if_not_installed_inquirer_with_option_show_failed_rules(capsys, client):
         captured = capsys.readouterr()
         assert captured.out == (
             '== The Rule IDs ==\n'
-            "'xccdf_org.ssgproject.content_rule_package_abrt_removed\\b'\n"
-            "You haven't got installed inquirer lib. Please copy id rule with you"
-            " want use and put it in command\n")
+            "^xccdf_org.ssgproject.content_rule_package_abrt_removed$\n"
+            "You haven't got installed inquirer lib. Please copy id rule with you want "
```

Comment:
I would suggest to use a regular expression here - you don't want to change tests every time that you slightly modify the message.